bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/FilterCriteria.java

```diff
@@ -1,155 +1,227 @@
- *
- * @author Kai Kreuzer - Initial contribution and API
- */
-public class FilterCriteria {
-
-    /** Enumeration with all possible compare options */
-    public enum Operator {
-        EQ("="), NEQ("!="), GT(">"), LT("<"), GTE(">="), LTE("<=");
-
-        private final String symbol;
-
-        Operator(String symbol) {
-            this.symbol = symbol;
-        }
-
-        String getSymbol() {
-            return symbol;
-        }
-    }
-
-    /** Enumeration with all ordering options */
-    public enum Ordering {
-        ASCENDING, DESCENDING
-    }
-
-    /** filter result to only contain entries for the given item */
-    private String itemName;
-
-    /** filter result to only contain entries that are newer than the given date */
-    private Date beginDate;
-
-    /** filter result to only contain entries that are older than the given date */
-    private Date endDate;
-
-    /** return the result list from starting index pageNumber*pageSize only */
-    private int pageNumber = 0;
-
-    /** return at most this many results */
-    private int pageSize = Integer.MAX_VALUE;
-
-    /** use this operator to compare the item state */
-    private Operator operator = Operator.EQ;
-
-    /** how to sort the result list by date */
-    private Ordering ordering = Ordering.DESCENDING;
-
-    /** filter result to only contain entries that evaluate to true with the given operator and state */
-    private State state;
-
-    public String getItemName() {
-        return itemName;
-    }
-
-    public Date getBeginDate() {
-        return beginDate;
-    }
-
-    public Date getEndDate() {
-        return endDate;
-    }
-
-    public int getPageNumber() {
-        return pageNumber;
-    }
-
-    public int getPageSize() {
-        return pageSize;
-    }
-
-    public Operator getOperator() {
-        return operator;
-    }
-
-    public Ordering getOrdering() {
-        return ordering;
-    }
-
-    public State getState() {
-        return state;
-    }
-
-    public FilterCriteria setItemName(String itemName) {
-        this.itemName = itemName;
-        return this;
-    }
-
-    public FilterCriteria setBeginDate(Date beginDate) {
-        this.beginDate = beginDate;
-        return this;
-    }
-
-    public FilterCriteria setEndDate(Date endDate) {
-        this.endDate = endDate;
-        return this;
-    }
-
-    public FilterCriteria setPageNumber(int pageNumber) {
-        this.pageNumber = pageNumber;
-        return this;
-    }
-
-    public FilterCriteria setPageSize(int pageSize) {
-        this.pageSize = pageSize;
-        return this;
-    }
-
-    public FilterCriteria setOperator(Operator operator) {
-        this.operator = operator;
-        return this;
-    }
-
-    public FilterCriteria setOrdering(Ordering ordering) {
-        this.ordering = ordering;
-        return this;
-    }
-
-    public FilterCriteria setState(State state) {
-        this.state = state;
-        return this;
-    }
-
-}
+/**
```

Comment:
I just noticed that the diff of this file looks rather large and also lines that are not changed are marked as changed. I think this might be a an issue with line endings ore file encoding because the indentation with spaces looks alright.

Could you please run the Eclipse formater again on this file?